24102 - update to wording used on button options.

TVWerdal opened the pull request:
Update to change text shown on button dropdown for Continuation filings

severinbeauvais reviewed (commented):
Are these any unit tests that need to be updated?

When ready to merge, ensure you are updating the app version (see other PRs for example).

JazzarKarim reviewed (commented):
Wait, are you working on this ticket here? https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/24102

The changes do not fix what the ticket is saying. It should be a different repo, not this one. The repo is: https://github.com/bcgov/business-dashboard-ui

TVWerdal commented:
> Wait, are you working on this ticket here? https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/24102
> 
> The changes do not fix what the ticket is saying. It should be a different repo, not this one. The repo is: https://github.com/bcgov/business-dashboard-ui

Yes, initially that was where I was looking, and I was confused because the wording is "Continuation Authorization" as it should be. However, this delete button is actually being generated in Action.vue in the in the business-registry-dashboard app on line 350. 

The const CONTINUATION_APPLICATION set in affiliation-types.ts is used in affiliations.ts to set the text for the const tempDescription. The only place this particular const is used currently is in Action.vue for the RemoveButton text online 350.

TVWerdal commented on business-registry-dashboard/app/enums/affiliation-types.ts:
```diff
@@ -1,7 +1,7 @@
   NAME_REQUEST = 'Name Request',
   AMALGAMATION_APPLICATION = 'Amalgamation Application',
-  CONTINUATION_APPLICATION = 'Continuation Application',
+  CONTINUATION_APPLICATION = 'Continuation Authorization',
```
Currently this const is only used once to set the tempDescription const in affiliations.ts. This is only used in one place, Action.vue on line 350 to set the text for the remove/delete button.

severinbeauvais replied:
See my other comment.

TVWerdal commented:
> Please update PR description to match our usual template (per my dm).
> 
> Are these any unit tests that need to be updated?
> 
> When ready to merge, ensure you are updating the app version (see other PRs for example).

This wasn't part of the tests previously, but I added one in to make sure it checks for any changes. I added a note there     
    // This is currently testing for the Delete button found in Action.vue, the only place this constant is used
    // If we require a constant for "Continuation Application" wording we will need to add logic to keep
    // desired wording for the delete button as outlined in https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/24102

severinbeauvais commented:
Thayne, this PR is the wrong fix as it is in the wrong UI.

Please look at the business-dashboard-ui code and make your changes there, and PR from there.

Looking at your comments, I think you'll need a second enum/constant for CONTINUATION_AUTHORIZATION and some code to switch between them depending on the filing's state.

TVWerdal commented:
Reassessing ticket and potential fix.
